_sass/screen.scss

```diff
@@ -32,6 +32,14 @@ $text-color: #636;
   }
 }
 
+
+// reused styling across top level containers
+@mixin desktop-container {
```

Comment:
I don't think I have one TBH - I think this explains what it's doing just fine 👍